_pages/robots.html

```diff
@@ -47,14 +47,14 @@ <h5>2015</h5>
             <p>Our 2015 robot, which we built for the <a href="https://www.youtube.com/watch?v=mAN1B7oKDXE">Recycle Rush</a> challenge, had the ability to stack totes quickly and effectively and manipulate recycling cans and trash, even in autonomous mode.</p>
         </div>
         <div onclick="location.href='robot/2014.html'" style="cursor:pointer;">
-            <a href="robot/2014.html"><img src="/images/robots/2014.png">
+            <img src="/images/robots/2014.png">
             <h5>2014</h5>
             <p>Our 2014 robot, which was designed for the <a href="https://www.youtube.com/watch?v=oxp4dkMQ1Vo">FIRST Aerial Assist</a> challenge, was able to catapult large balls with 100% consistency.</p>
-        </div>
-        <div>
+        </div>    
+        <div onclick="location.href='robot/2013.html'" style="cursor:pointer;">
```

Comment:
Any way we can just make this an `<a>` tag?

Reply:
With this, we can click anywhere inside the `<div>`, whereas with an `<a>` tag only the title and image are clickable.

Reply:
Hmm, try adding a `background` style on the relevant component and it should work. I think that would be preferable to adding inline CSS *and* JS on this element

Reply:
Wait, I see we do this elsewhere—that is not great, not sure when that made it in. It would be cool if you could clean that up, but not necessary for this PR.